universal-login-react/src/ui/Transfer/Amount/ModalTransferAmount.tsx

```diff
@@ -6,7 +6,7 @@ import {getStyleForTopLevelComponent} from '../../../core/utils/getStyleForTopLe
 
 export interface ModalTransferAmountProps {
   sdk: UniversalLoginSDK;
-  ensName: string;
+  contractAddress: string;
```

Comment:
Use `DeployedWallet` instead, it already contains `sdk`, `ensName` and `contractAddress`.

You can pass it `ModalTransfer`, where it is wrongly names `ApplicationWallet`.
Good opportunity to rename.

So that you will have:
```ts
<ModalTransferAmount deployedWallet={deployedWallet} onSelectRecipientClick={...} updateTransferDetailsWith={...} currency={...}/>
```

No need for `sdk` and `contractAddress`/`ensName`.

Reply:
#useDeployedWallet